src/core/core.datasetController.js

```diff
@@ -980,6 +980,10 @@ helpers.extend(DatasetController.prototype, {
 	 * @private
 	 */
 	_getSharedOptions: function(mode, el, options) {
+		if (!mode) {
+			// store element option sharing status for usage in interactions
+			this._elementsShareOptions = options && options.$shared;
```

Comment:
How about just making this variable name be `_sharedOptions`?

Reply:
done